api/axis.py

```diff
@@ -1247,20 +1277,12 @@ def process_one(self, e, namespaces):
             expiration_date = datetime.strptime(
                 expiration_date, self.FULL_DATE_FORMAT)
 
-        # WTH??? Why is identifier None? Is this ever used?
-        fulfillment = FulfillmentInfo(
-            collection=self.collection, data_source_name=DataSource.AXIS_360,
-            identifier_type=self.id_type,
-            identifier=None, content_link=fulfillment_url,
-            content_type=None, content=None, content_expires=None
-        )
```

Comment:
This was not, in fact, ever used. We never fulfill a book immediately after checking it out. I removed this code rather than make it work in a world where an `EbookFulfillmentInfo` is expected to contain a book vault ID. That's because a checkout response document doesn't seem to _have_ a book vault ID -- it's only found in an availability document, which you can obtaine _after_ the checkout..